src/client/components/ConsentsSubHeader.tsx

```diff
@@ -78,8 +78,9 @@ const li = (numPages: number) => css`
     ${textSans.small()}
   }
   position: relative;
-  width: ${100 / numPages}%;
+  width: ${lastPage ? 'min-content' : 100 / (numPages - 1) + '%'};
   display: flex;
+  flex-grow: ${lastPage ? '0' : '1'};
```

Comment:
Similarly to the line above, this ensures that the last column doesn't grow and cause the steps to shift to the left